frontend/src/utils/api.ts

```diff
@@ -239,13 +244,15 @@ window.api = {
 
   async loadCommits(repoName) {
     const folderName = window.REPOS[repoName].outputFolderName;
-    const commits = await this.loadJSON(`${folderName}/commits.json`);
-    const res = [];
+    const json = await this.loadJSON(`${folderName}/commits.json`);
+    const commits = commitsSchema.parse(json);
+
+    const res: User[] = [];
     const repo = window.REPOS[repoName];
 
     Object.keys(commits.authorDisplayNameMap).forEach((author) => {
       if (author) {
-        const obj = {
+        const obj: any = {
```

Comment:
Is it possible to just declare a `user` and fill up the fields and push it to `res`? That way we don't need to type it to `any` then covert to `User` again

Reply:
Yup thanks, just pushed a commit for this! I originally used `any` as I wanted to keep the code as similar as possible and work on a bigger refactor to remove the `any`s (+ in other .ts files as well), but I think it makes sense to do a tiny refactor here to avoid introducing a redundant typecast that will be removed again in the future.